packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -112,3 +112,24 @@ Template locking allows locking the `InnerBlocks` area for the current template.
 If locking is not set in an `InnerBlocks` area: the locking of the parent `InnerBlocks` area is used.
 
 If the block is a top level block: the locking of the Custom Post Type is used.
+
+### `renderAppender`
+* **Type:** `String|Function`
```

Comment:
In our new format I think renderAppender may simply be an element instead of a function:
e.g:
```
renderAppender={
	<InnerBlocks.HideWhenChildren>
		<InnerBlocks.ButtonAppender />
	</InnerBlocks.HideWhenChildren>
}
```
Instead of:
```
renderAppender={ () => (
	<InnerBlocks.HideWhenChildren>
		<InnerBlocks.ButtonAppender />
	</InnerBlocks.HideWhenChildren>
) }
```

Maybe we can just call it appender?

Reply:
@jorgefilipecosta Don't we want to allow users to supply their own render function? I guess users could supply their own components instead but then we lose any ability in future to be able to pass arguments to the `render` (which a function affords but not an element).